app/src/main/res/layout/editor_bottom_sheet.xml

```diff
@@ -45,10 +46,12 @@
                 android:id="@+id/et_basic_editor"
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:layout_marginLeft="8dp"
                 android:dropDownAnchor="@id/basic_editor_scroll"
                 android:inputType="textMultiLine|textCapSentences"
-                android:minHeight="48dp"
+                android:minHeight="46dp"
```

Comment:
Why this change? 48dp is the minimum size of a touchable item in the UI guidelines.

Reply:
Didn't know about the guideline.
I've changed it to make the bottom sheet TextView look a bit more centered and aligned with the nearby buttons after reducing text size to 16sp, but the difference is actually very subtle. I'll let you choose :)

Reply:
I'd prefer to stay with standard sizes, I think.

Reply:
For "stay with standard sizes" you mean only for the `minHeight` or for both `textSize` and `minHeight`?

Reply:
I was referring to `minHeight` only.